docs/concepts/abi_overview.md

```diff
@@ -204,13 +203,14 @@ typedef struct TVMFFIObject {
 
 `TVMFFIObject` defines a common 24-byte intrusive header that all in-memory objects share:
 
-- `strong_ref_count` stores the strong atomic reference counter of the object.
-- `weak_ref_count` stores the weak atomic reference counter of the object.
+- `combined_ref_count` packs strong and weak reference counter of the object into a single 64bit field
+  - The lower 32bits stores the strong atomic reference counter
+  - The higher 32bits stores the weak atomic reference counter
```

Comment:
do we also want to talk about how to obtain the lower/higher 32bits? e.g. `combined_ref_count & kCombinedRefCountMaskUInt32`